app/utils/csv_files.py

```diff
@@ -0,0 +1,34 @@
+import os
+import uuid
+from pathlib import Path
+
+import pandas as pd
+from fastapi import UploadFile
+
+from app.schemas import CSVMetadataCreate
+
+
+async def save_uploaded_csv(file: UploadFile, dir: Path):
+    name = uuid.uuid4().hex
```

Comment:
IMO it's better to rely on the internal `__str__` implementation of an object:
```suggestion
    name = str(uuid.uuid4())
```

Reply:
✅ Resolved in [2bfcb67](https://github.com/ttnsy/fastapi-csv-upload/pull/2/commits/2bfcb67713c0285c7b92a624dd75ae0caf1b0980)